code/modules/antagonists/changeling/powers/gaseous_pores.dm

```diff
@@ -1,27 +1,27 @@
 /datum/action/changeling/gaseous_pores
 	name = "Gaseous Pores"
-	desc = "Our skins bursts, releasing somniferous gases to put opponents to sleep and cover our retreat."
+	desc = "Our skins bursts, releasing gases which bring confusion to our chasers, covering our escape."
```

Comment:
```suggestion
	desc = "Our skins bursts, releasing gases which bring confusion to our pursuers, covering our escape."
```
might just be me but "chasers" sounds a bit weak of a word

Reply:
Yeah, that it does